ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -527,6 +527,13 @@ public void setAutoCapitalize(ReactEditText view, int autoCapitalize) {
 
   @ReactProp(name = "keyboardType")
   public void setKeyboardType(ReactEditText view, @Nullable String keyboardType) {
+    int passwordVisibilityFlag = InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD &
```

Comment:
So this flag is only the visibility bit if I understand correctly? It won't unset the fact that it is a password or not.

Reply:
That is correct. We wouldn't want to set/unset general password status in `keyboardType`, only in `secureTextEntry`.